Data/SourceAssets/Localization/English/IngameTrainingInstructions.txt

```diff
@@ -1,14 +1,14 @@
 Enter the room right in front of you.
-Stand right in front of the door.\Press �Return
+Stand right in front of the door.\Press ûReturn.
```

Comment:
The diff shows that the special character before the "Return" has changed. I assume this happened by re-saving the file?
I suppose that character changed the text color, so maybe that does not work anymore?

Reply:
True, Github changed the encoding to UTF-8. Checking in from Source again

Reply:
Is there any github setting we should change for this repo?

Reply:
I am not aware of any setting that would allow to change the default encoding of Githubs Web-Editor. Google comes up empty as well. (same question but no answers). As we are now aware of the fact, it shouldn't be a big thing.

Reply:
@ungesundes-halbwissen , I changed the encoding rule for .txt in the .gitattributes - I hope this will prevent it from happening again